src/util.ts

```diff
@@ -0,0 +1,9 @@
+export const removeNullFields = (obj: any) => Object.keys(obj)
+  .filter((key) => obj[key] !== null && obj[key] !== undefined)
```

Comment:
the check can be condensed into:

```
obj[key] != null
```
https://stackoverflow.com/a/55539154/2657249

Reply:
Refactored